.github/workflows/portable-rustls-ci-build.yml

```diff
@@ -111,14 +116,22 @@ jobs:
         env:
           RUSTFLAGS: --cfg portable_atomic_unstable_coerce_unsized --cfg unstable_portable_atomic_arc
 
-      - name: cargo test (debug; all features)
+      - name: cargo test (debug; unstable-std-extra [should cover all non-unsafe features])
+        run: cargo test -F unstable-std-extra --all-targets
+        working-directory: rustls
+        env:
+          RUSTFLAGS: --cfg portable_atomic_unstable_coerce_unsized --cfg unstable_portable_atomic_arc
+          RUSTDOCFLAGS: --cfg portable_atomic_unstable_coerce_unsized --cfg unstable_portable_atomic_arc
```

Comment:
need comment with explanation if both of these env settings are really required for this to work ... also find a way to avoid the repetition here, if possible